app.py

```diff
@@ -52,7 +52,60 @@ def mirror(name):
     return create_response(data)
 
 
-# TODO: Implement the rest of the API here!
+@app.route("/users")
+def get_users():
+    data = db.get('users')
+
+    if "team" in request.args:
+        data = list(filter(lambda x: x["team"] == request.args["team"], data))
+
+    return create_response({"users": data})
+
+
+@app.route("/users/<id>")
+def get_user(id):
+    data = db.getById('users', int(id))
```

Comment:
can you find a way to avoid the repeated casting `int(id)`? (hint: try to get flask to do it automatically for you)